pilot/cmd/pilot-discovery/app/cmd.go

```diff
@@ -153,7 +153,7 @@ func addFlags(c *cobra.Command) {
 	c.PersistentFlags().StringVar((*string)(&serverArgs.RegistryOptions.KubeOptions.ClusterID), "clusterID", features.ClusterName,
 		"The ID of the cluster that this Istiod instance resides")
 	c.PersistentFlags().StringToStringVar(&serverArgs.RegistryOptions.KubeOptions.ClusterAliases, "clusterAliases", map[string]string{},
-		"Alias names for clusters")
+		"Alias names for clusters. Example: alias1=cluster1,alias2=cluster2")
```

Comment:
I am not sure if or where we document `extraContainerArgs` or provide an example like this in the Istio documentation. It would be great to follow up this PR with a docs PR adding more documentation for this use case!

Reply:
If I right understood the docs section is autogenerated from cmd code.
https://github.com/istio/istio.io/blob/86b126b38c777e9c645fc96838ab8c931f05c23d/content/zh/docs/reference/commands/pilot-discovery/index.html#L179-L181

Reply:
Okay, great. I don't think `extraContainerArgs` is documented though.